lib/settings.js

```diff
@@ -35,9 +36,15 @@ async function plistPaths (sim, identifier) {
       paths.push(path.resolve(sim.getDir(), 'Library', 'Preferences', 'com.apple.locationd.plist'));
       break;
     case 'userSettings':
-      paths.push(path.resolve(sim.getDir(), 'Library', 'ConfigurationProfiles', 'UserSettings.plist'));
-      paths.push(path.resolve(sim.getDir(), 'Library', 'ConfigurationProfiles', 'EffectiveUserSettings.plist'));
-      paths.push(path.resolve(sim.getDir(), 'Library', 'ConfigurationProfiles', 'PublicInfo', 'PublicEffectiveUserSettings.plist'));
+      if (compare(sim.xcodeVersion.versionString, '7.3') < 0) {
```

Comment:
I think this sort of optimization is performed by the compiler. I'm not so sure making the change improves readability either.

Made the change anyways since it's minor :)